src/components/forms/SelectField/SelectField.stories.jsx

```diff
@@ -63,47 +64,20 @@ export const Static = {
   },
 };
 
-export const MenuOpen = {
-  name: 'Force menu open',
-  args: {
-    name: 'select',
-    id: 'select-static',
-    label: 'Static options',
-    description: 'This is a custom description for the select field',
-    disabled: false,
-    isRequired: true,
-    options: [
-      {value: 'option-1', label: 'Option 1'},
-      {value: 'option-2', label: 'Option 2'},
-    ],
-    menuIsOpen: true,
-  },
-  parameters: {
-    formik: {
-      initialValues: {
-        select: '',
-      },
-    },
-  },
-};
-
-const delay = async delay => await new Promise(resolve => setTimeout(resolve, delay));
-
 export const Async = {
   render: function Render({getOptionsDelay, dynamicOptions, ...args}) {
     const serializedOptions = JSON.stringify(dynamicOptions);
     const getOptions = useCallback(async () => {
-      await delay(getOptionsDelay);
+      await sleep(getOptionsDelay);
```

Comment:
Is sleep used for more general purposes and delay for user events like typing?This is what I understood.